### ISSUE TITLE
all checked points report listing, as a result of my evaluation

### ISSUE THREAD

**Issue by stenlisuryadinata** [LarryIVC/portfolio#14]
all checked points report listing, as a result of my evaluation, are as follows: 

Page titles - no issue found
image text alternatives - no issue found
text headings - no issue found 
color contrast - have an issue, please check below
resize - no issue found
interaction - no issue found 
moving content - no issue found 
multimedia - no issue found 
the basic structure of the page - no issue found. 

everything is good, the only remarks are as follows: 

1. the logo's  contrast (4.44:1) is low
the contrast between text and background colors is very low

https://github.com/LarryIVC/portfolio/blob/2629e4e904fd77393bfbb54704a331d9403337af/index.html#L16

2. the let's connect contrast (2.95:1) is low
the contrast between text and background colors is very low

https://github.com/LarryIVC/portfolio/blob/2629e4e904fd77393bfbb54704a331d9403337af/index.html#L38

4. the back-end contrast (3.68:1) is low
the contrast between text and background colors is low
https://github.com/LarryIVC/portfolio/blob/2629e4e904fd77393bfbb54704a331d9403337af/index.html#L58

5. the HTML, CSS, and javascript text  contrast (3.39:1) is low
https://github.com/LarryIVC/portfolio/blob/2629e4e904fd77393bfbb54704a331d9403337af/index.html#L64

6. the contact form accessibility is  Missing the form label
 the contact form does not have a corresponding label for a form control 

https://github.com/LarryIVC/portfolio/blob/2629e4e904fd77393bfbb54704a331d9403337af/index.html#L196

7. the text contrast (3.39:1) is low
the contrast between text and background colors is low. 

https://github.com/LarryIVC/portfolio/blob/2629e4e904fd77393bfbb54704a331d9403337af/index.html#L192

8. the text area missing form label
the text area does not have a corresponding label for form control 

Thank you

**Comment by LarryIVC** [comment]
Thanks, @stenlisuryadinata 👏  for your collaboration 🛸  in this project.
Happy Coding 🥳
